playwright-e2e/dsm/component/tables/participantListTable.ts

```diff
@@ -1,14 +1,33 @@
 import { Locator, Page } from '@playwright/test';
 import Table from 'dss/component/table';
 import ParticipantPage from 'dsm/pages/participant-page/participant-page';
+import {ParticipantsListPaginator} from 'lib/component/dsm/paginators/participantsListPaginator';
+import {rows} from 'lib/component/dsm/paginators/types/rowsPerPage';
 
 export class ParticipantListTable extends Table {
```

Comment:
One more thing, files names should follow this convention `participant-list-table.ts`. Not a big deal, it can be refactored in future.